tests/dns.rs

```diff
@@ -0,0 +1,92 @@
+use std::{
+    fs, io,
+    io::BufRead,
+    path::PathBuf,
+    process::{Child, Command},
+    sync::mpsc::{self, Receiver, Sender},
+    thread,
+    time::Duration,
+};
+
+use coinswap::{
+    maker::error::MakerError,
+    market::rpc::{read_resp_message, RpcMsgReq, RpcMsgResp},
+    utill::send_message,
+};
+
+use tokio::{io::BufReader, net::TcpStream};
```

Comment:
Can you do this with std Buffreader and TcpStream? We don't need tokio here and will remove it soon.